Serializer/JsonLDNormalizer.php

```diff
@@ -66,6 +70,10 @@ public function supportsNormalization($data, $format = null, array $context = []
 
     public function supportsDenormalization($data, $type, $format = null, array $context = []) : bool
     {
+        if (!$this->isCorrectFormat($format)) {
```

Comment:
Wouldn't this interfere more when tolerating $format == null? Before we didn't as far as I understand

Reply:
In this method? We absolutely did, there is no check for it. In `supportsNormalization`, we previously would only accept a null format if it was an array and not an object. I thought about this for a bit, and changed the check to only accept `null` if the thing being (de)normalised is an array. That better matches what was done before.